:func:`posixpath.commonpath()` now raises a :exc:`ValueError` exception when
passed an empty iterable. Previously, :exc:`IndexError` was raised.

:func:`posixpath.commonpath()` now raises a :exc:`TypeError` exception when
passed ``None``. Previously, :exc:`ValueError` was raised.
